File: internal/model/tenant.go
package model

import "time"

type Tenant struct {
	ID        string    `json:"id"`
	FirstName string    `json:"first_name"`
	LastName  string    `json:"last_name"`
	Address   string    `json:"address"`
	Email     string    `json:"email"`
	Phone     string    `json:"phone"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}

File: web/src/tests/msw/factories/tenant.ts
import type { TTenant } from '@/domains/tenants';

export const createMockTenant = (overrides: Partial<TTenant> = {}): TTenant => ({
  id: 't1',
  first_name: 'Alex',
  last_name: 'Doe',
  email: 'alex@example.com',
  phone: '555-0100',
  created_at: '2026-01-01T00:00:00Z',
  updated_at: '2026-01-01T00:00:00Z',
  ...overrides,
});
